### ISSUE TITLE
cannot import name 'DEFAULT_CIPHERS' from 'urllib3.util.ssl_

### ISSUE THREAD

**Issue by leo-mercier** [t4d/StalkPhish#40]
Hello,
i have this error, can you help me.

2023-05-30 16:24:07,135 - StalkPhish.py - INFO - Configuration file to use: .\stalkphish\conf\example.conf
2023-05-30 16:24:07,135 - StalkPhish.py - INFO - Database: ./db/StalkPhish.sqlite3
2023-05-30 16:24:07,136 - StalkPhish.py - INFO - Main table: StalkPhish
2023-05-30 16:24:07,158 - StalkPhish.py - INFO - Investigation table: StalkPhishInvestig
2023-05-30 16:24:07,166 - StalkPhish.py - INFO - Files directory: ./files/
2023-05-30 16:24:07,167 - StalkPhish.py - INFO - Download directory: ./dl/
2023-05-30 16:24:07,167 - StalkPhish.py - INFO - Declared Proxy: None

2023-05-30 16:24:07,167 - StalkPhish.py - INFO - Proceeding to OSINT modules launch
2023-05-30 16:24:07,547 - urlscan.py - INFO - Searching for 'cliquesante'...
2023-05-30 16:24:08,351 - urlquery.py - INFO - Searching for 'cliquesante'...
2023-05-30 16:24:08,358 - StalkPhish.py - ERROR - Main error (<class 'ImportError'>, ImportError("cannot import name 'DEFAULT_CIPHERS' from 'urllib3.util.ssl_' (C:\\Users\\merci\\PycharmProjects\\StalkPhish\\venv\\lib\\site-packages\\urllib3\\util\\ssl_.py)"), <traceback object at 0x000001E7AB431180>)

**Comment by t4d** [comment]
Indeed, it may be necessary to modify the code, particularly because of the new urllibv3 features. I'll look into it.

**Comment by leo-mercier** [comment]
thanks ! I tried to change some things but no results

**Comment by t4d** [comment]
I made tests with a Linux Alpine 3.18 and urllib3 v2.0.2, and this works as it should. What's your urllib3 version? Did you try to upgrade your urllib3 package?

**Comment by nfour14** [comment]
Hi! I am getting a similar error - urllib3 2.0.3 is what I am using, this is on Ubuntu 22.04. Maybe I need a downgrade of urllib3?

2023-07-08 18:31:58,976 - StalkPhish.py - ERROR - Main error (<class 'ImportError'>, ImportError("cannot import name 'DEFAULT_CIPHERS' from 'urllib3.util.ssl_' (/home/user/.pyenv/versions/StalkPhishProj/lib/python3.10/site-packages/urllib3/util/ssl_.py)"), <traceback object at 0x7f13e28ea8c0>)

**Comment by nfour14** [comment]
Downgrade to urllib3 version 1.26.16 seemed to get rid of that error.
